liquibase/changelog/changelog.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <changeSet id="create-user-table" author="Vansh">
+        <createTable tableName="User">
+            <column name="UserID" type="INT" autoIncrement="true">
```

Comment:
Column name should be just "id" not UserID as it is primary key.

Reply:
ok i will change it

Reply:
change done